packages/botonic-cli/src/commands/train.ts

```diff
@@ -38,6 +38,16 @@ class Task {
   }
 }
 
+class InvalidTaskError extends Error {
```

Comment:
What do you think about creating a specific error for this? :thinking: @dpinol @vanbasten17

Reply:
Whatever you decide, be consistent. For instance, it doesn't look to consistent that logTaskPluginNotInstalled generates the error message in a function, but InvalidTaskError does it in an exception